src/reference/00-Getting-Started/05-Basic-Def.md

```diff
@@ -138,8 +97,8 @@ the value `"hello"`.
 If you use the wrong value type, the build definition will not compile:
```

Comment:
Yes, I also find the immutability break dancing to be odd. I'll remove the mention of it.

To be fair, the original authors might be saying that even though the DSL exposes procedural interface (assigning to a variable-like thing), *internally* it's kept purely functional by translating the DSL to something like:

```scala
scala> Map.empty[Symbol, Int].updated('x, 1).updated('x, 2)
res0: scala.collection.immutable.Map[Symbol,Int] = Map('x -> 2)
```

I don't see how that can be relevant to sbt users.